feature-group-definitions/font-synthesis-small-caps.yml

```diff
@@ -0,0 +1,12 @@
+name: font-synthesis-small-caps
+description: "The `font-synthesis-small-caps` CSS property sets whether or not the browser should synthesize small caps typefaces when they're missing from the font."
+spec: https://drafts.csswg.org/css-fonts-4/#font-synthesis-small-caps
+group: font-synthesis
+usage_stats: https://chromestatus.com/metrics/css/timeline/popularity/703
+compat_features:
+  - css.properties.font-synthesis-small-caps
```

Comment:
There are `auto` and `none` subfeatures here. Should we include them too? They look to have the same support values.